### ISSUE TITLE
QSelect 1.0 Don't work create new values without Chips

### ISSUE THREAD

**Issue by natxocc** [quasarframework/quasar#2945]
### Software version

Quasar: 1.0 alfa testing
OS: Windows 10
Node: 10
NPM:
Browsers: Chrome 71
iOS:
Android:
Any other software related to your bug:

### What did you get as the error?
When I use creating new values in Qselect 1.0, work correctly with Chips, but without them, only the text, the input value is cleared when exit the focus

### What were you expecting?
The value I wrote need to be on input after exiting focus to other component

### What steps did you take, to get the error?
Using the Quasar 1.0 alfa with updated until 2019-01-02

**Comment by natxocc** [comment]
Yes, With ENTER key and Chips is working well as I said. The question is, how to do to write a Input text (simple text) without chips? It's not working (without TAB, without ENTER, without anything... always clear the input value when lost focus)

**Comment by rstoenescu** [comment]
It's not designed to work without the "multiple" prop, but will add support for that too in 1.0.

**Comment by natxocc** [comment]
Thanks Razvan! This option is very usefull for me when I want to insert new records in database and use QSelect to decide the user if want from a list or create a new one.
